installer/resources/lambda_rule_engine/files/rule_engine_cloudwatch_azure_rules.json

```diff
@@ -2621,6 +2621,28 @@
       "modifiedDate": "2022-08-23",
       "severity": "medium",
       "category": "security"
-    }
-  ]
+    },
+  {
+    "ruleId": "Enable_latest_TLS_version",
+    "ruleUUID": "Enable_latest_TLS_version",
+    "policyId": "Enable_latest_TLS_version",
+    "ruleName": "Enable storage account min Tls version set to 1.2",
+    "targetType": "storageaccount",
+    "assetGroup": "azure",
+    "alexaKeyword": "Enable_latest_TLS_version",
+    "ruleParams": "{\"params\":[{\"encrypt\":false,\"value\":\"check-for-latest-Tls-version-rule\",\"key\":\"ruleKey\"},{\"key\":\"dataBaseType\",\"value\":\"SQLSERVER\",\"isValueNew\":true,\"encrypt\":false},{\"key\":\"minimumTlsVersion\",\"value\":\"TLS1_2\",\"isValueNew\":true,\"encrypt\":false},{\"key\":\"policyName\",\"value\":\"Enable storage account min Tls version set to 1.2\",\"isValueNew\":true,\"encrypt\":false},{\"encrypt\":false,\"value\":\"medium\",\"key\":\"severity\"},{\"encrypt\":false,\"value\":\"security\",\"key\":\"ruleCategory\"},{\"encrypt\":false,\"value\":\"\",\"key\":\"ruleOwner\"}],\"environmentVariables\":[],\"ruleId\":\"Enable_latest_TLS_version\",\"autofix\":false,\"alexaKeyword\":\"Enable_latest_TLS_version\",\"ruleRestUrl\":\"\",\"targetType\":\"storageaccount\",\"pac_ds\":\"azure\",\"policyId\":\"Enable_latest_TLS_version\",\"assetGroup\":\"azure\",\"ruleUUID\":\"Enable_latest_TLS_version\",\"ruleType\":\"ManageRule\"}",
+    "ruleFrequency": "0 0/6 * * ? *",
+    "ruleExecutable": "",
+    "ruleRestUrl": "",
+    "ruleType": "ManageRule",
+    "ruleArn": "arn:aws:events:us-east-1:***REMOVED***:rule/Enable_latest_TLS_version",
+    "status": "ENABLED",
+    "userId": "ASGC",
+    "displayName": "Enable storage account min Tls version set to 1.2",
```

Comment:
Modify the rule name to use latest instead of V1.2

Reply:
changed rule name to use latest instead of v1.2.